CONTRIBUTING.md

```diff
@@ -28,6 +28,7 @@ We've provided a list of community-driven content below. When adding content to
 - [Recurring Events](#content-recurring-event)
 - [Upcoming Events](#content-upcoming_event)
 - [The OCaml Changelog](#content-changelog)
+- [PR Template](#content-cookbook)
```

Comment:
I would recommend to put this under the existing "OCaml Cookbook Recipes" section, not as a separate section.

Overall looks useful to me. :+1: